### ISSUE TITLE
Test nedelec_non_rect_face.debug runs way too long

### ISSUE THREAD

**Issue by bangerth** [dealii/dealii#642]

**Comment by bangerth** [comment]
The test `nedelec_non_rect_face.debug` runs about 78 minutes on my system without load, so it fails every single time because we have a 10 minute timeout limit. I suppose that's the case for everyone else as well.

We should see if we can reduce the run-time of this somehow, or to split it up into more than one test so that they can run in parallel.

This came in through a patch by Ross Kynch in #197

**Comment by rosskynch** [comment]
It's due to the length of time to call the constructor for FE_Nedelec in debug mode and also the large linear system which needs solving for the higher p. I'll reduce it to p=3 and start the mesh refinement from 1 element instead of 8.

**Comment by rosskynch** [comment]
Removing the p=3 seems to quicken it up a lot.
[100%] Run nedelec_non_rect_face with Debug configuration
[100%] Built target run
real	1m13.862s
user	1m6.778s
sys	0m0.166s

It's still enough to see it's working, so I'll do a pull req now.

**Comment by bangerth** [comment]
Fantastic -- many thanks!

**Comment by bangerth** [comment]
Fixed by #645
